ceph-defaults/README.md

```diff
@@ -0,0 +1,3 @@
+# Ansible role: ceph-defaults
+
+Documentation is available at http://docs.ceph.com/cephadm-ansible/.
```

Comment:
i assume this link will actually go to some documentation in the future but for now should this dead link be included?

Reply:
Yeh, just tried to anticipate when the doc will be built/available. Not a big issue IMO, we can leave it as it is for now and update it later if needed I guess.